blockchain/chainstore.go

```diff
@@ -102,6 +99,7 @@ func (c *ChainStore) loop() {
 			}
 
 		case closed := <-c.quit:
+			c.IStore.Close()
```

Comment:
Different from the dev branch implementation.

Reply:
fixed